doc/design/OrionHDFSSink.md

```diff
@@ -24,27 +24,37 @@ Independently of the data generator, NGSI context data is always [transformed](f
+    * `json-row`: A JSON line is added for each notified context attribute. This kind of line will always contain 8 fields:
         * `recvTimeTs`: UTC timestamp expressed in miliseconds.
         * `recvTime`: UTC timestamp in human-redable format ([ISO 6801](http://en.wikipedia.org/wiki/ISO_8601)).
         * `entityId`: Notified entity identifier.
         * `entityType`: Notified entity type.
         * `attrName`: Notified attribute name.
         * `attrType`: Notified attribute type.
-        * `attrValue`: In its simplest form, this value is just a string, but since Orion 0.11.0 it can be Json object or Json array.
-        * `attrMd`: It contains a string serialization of the metadata array for the attribute in Json (if the attribute hasn't metadata, an empty array `[]` is inserted).
-    * `column`: A single Json line is added for all the notified context attributes. This kind of line will contain two fields per each entity's attribute (one for the value, called `<attrName>`, and other for the metadata, called `<attrName>_md`), plus an additional field about the reception time of the data (`recvTime`).
+        * `attrValue`: In its simplest form, this value is just a string, but since Orion 0.11.0 it can be JSON object or JSON array.
+        * `attrMd`: It contains a string serialization of the metadata array for the attribute in JSON (if the attribute hasn't metadata, an empty array `[]` is inserted).
+    * `json-column`: A single JSON line is added for all the notified context attributes. This kind of line will contain two fields per each entity's attribute (one for the value, called `<attrName>`, and another for the metadata, called `<attrName>_md`), plus an additional field about the reception time of the data (`recvTime`).
+    * `csv-row`: A CSV line is added for each notified context attribute. As `json-row`, this kind of line will always contain 8 fields:
+        * `recvTimeTs`: UTC timestamp expressed in miliseconds.
+        * `recvTime`: UTC timestamp in human-redable format ([ISO 6801](http://en.wikipedia.org/wiki/ISO_8601)).
+        * `entityId`: Notified entity identifier.
+        * `entityType`: Notified entity type.
+        * `attrName`: Notified attribute name.
+        * `attrType`: Notified attribute type.
+        * `attrValue`: In its simplest form, this value is just a string, but since Orion 0.11.0 it can be JSON object or JSON array.
+        * `attrMd`: In this case, the field does not contain the real metadata, but the name of the HDFS file storing such metadata. The reason to do this is the metadata may be an array of any length; each element within the array will be persisted as a single line in the metadata file. There will be a metadata file per each attribute under `/user/<hdfs_userame>/<fiware-service>/<fiware-servicePath>/<destination>_<attrName>_<attrType>/<destination>_<attrName>_<attrType>.txt`
```

Comment:
The format of the metadata lines in the.txt files needs to be explained, e.g. how the metadata name, type and value are represented there, if recvTimeTs/recvTime are used in that line, etc.